### PR TITLE
chore: updated readme

### PR DESCRIPTION
**Pull request by DharaniDJ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
...
+        docker build -t fuseki .
+        ```
+     b. Run the Docker image:
+        ```
+        docker run -p 3030:3030 fuseki
+        ```
+
+3. **Run the Spring Boot Application:**
+   - Change directory to `Application Server`.
+   - Execute `gradle.bat bootRun`.
```
Comment (vgnshiyer) [inline README.md:40]: Add one for mac version as well

`./gradlew bootRun`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,59 @@
...
+
+4. **Local Configuration:**
+   - Since the application is running locally, replace all occurrences of `datastore` with `localhost` in the configuration.
```
Comment (vgnshiyer) [inline README.md:43]: This is not required.. You can remove this.. We are running backend and fuseki via docker compose

File: README.md [hunk 3]
```diff
@@ -1,2 +1,59 @@
...
+
+5. **Accessing the Data:**
+   - Access the data using the APIs. Refer to the `controllers` folder for the endpoints.
```
Comment (vgnshiyer) [inline README.md:46]: Remove this

**Comment by vgnshiyer** [comment]
Give the controllers to chat gpt and ask it to provide api docs in markdown format.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,56 @@
...
+        ```
+
+3. **Run the Spring Boot Application:**
+   - For Windows:
+     - Change directory to `Application Server`.
+     - Execute `gradle.bat bootRun`.
+   - For Mac:
+     - Change directory to `Application Server`.
+     - Run `./gradlew bootRun`.
+
```
Comment (DevanshuDesai15) [inline README.md:43]: Can you also add "Setup for Frontend" 
First go to the WebApp directory and then do npm install and then npm run dev command to start the frontend

File: README.md [hunk 5]
```diff
@@ -1,2 +1,278 @@
...
+- **Description**: Retrieves victims related to a specific crime event.
+
+#### 9. Get Perpetrators by Crime Event
+- **Endpoint**: `/{id}/perpetrators`
+- **Method**: `GET`
+- **Path Variables**:
+  - `id` (integer): The ID of the crime event.
+- **Description**: Retrieves perpetrators related to a specific crime event.
+
+#### 10. Create Crime Event
```
Comment (vgnshiyer) [inline README.md:126]: Let's remove this..

File: README.md [hunk 6]
```diff
@@ -1,2 +1,278 @@
...
+- **Description**: Retrieves perpetrators filtered by race.
+
+#### 5. Get Perpetrators by Gender
+- **Endpoint**: `/gender/{gender}`
+- **Method**: `GET`
+- **Path Variables**:
+  - `gender` (string): The gender of the perpetrators.
+- **Description**: Retrieves perpetrators filtered by gender.
+
+#### 6. Update Perpetrator by ID
```
Comment (vgnshiyer) [inline README.md:187]: This one as well

File: README.md [hunk 7]
```diff
@@ -1,2 +1,278 @@
...
+- **Description**: Retrieves victims filtered by race.
+
+#### 5. Get Victims by Gender
+- **Endpoint**: `/gender/{gender}`
+- **Method**: `GET`
+- **Path Variables**:
+  - `gender` (string): The gender of the victims.
+- **Description**: Retrieves victims filtered by gender.
+
+#### 6. Update Victim by ID
```
Comment (vgnshiyer) [inline README.md:248]: rm -rf

**Pull request merged**